libs/cdk/src/lib/template-management/model.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  ChangeDetectorRef,
+  ElementRef,
+  EmbeddedViewRef,
+  NgIterable,
+  NgZone,
+  TemplateRef,
+  ViewContainerRef
+} from '@angular/core';
+import { Observable } from 'rxjs';
+import { StrategyCredentialsMap } from '../render-strategies/model';
+
+export type rxBaseTemplateNames = 'rxError' | 'rxComplete' | 'rxSuspense';
+
+export enum RxBaseTemplateNames {
+  error = 'rxError',
+  complete = 'rxComplete',
+  suspense = 'rxSuspense'
+}
+
+export const enum ListChange {
+  insert,
+  remove,
+  move,
+  update,
+  context,
+}
```

Comment:
https://github.com/ReactiveX/rxjs/blob/master/src/internal/Notification.ts
looking here it says: `NotificationKind is deprecated as const enums are not compatible with isolated modules. Use a string literal instead.`

Reply:
If we use it only folder internal it should not matter IMHO

Reply:
I saw u changed it. close this here...